test/end2end_test.go

```diff
@@ -1354,7 +1354,7 @@ func interestingGoroutines() (gs []string) {
 	return
 }
 
-var failOnLeaks = flag.Bool("fail_on_leaks", false, "Fail tests if goroutines leak.")
+var failOnLeaks = flag.Bool("fail_on_leaks", true, "Fail tests if goroutines leak.")
```

Comment:
Maybe just remove this flag entirely? I can't imagine when you'd want to run without it.